ui/scripts/system.js

```diff
@@ -9996,7 +9996,7 @@
                                             }
 
                                             $.ajax({
-                                                url: createURL("listRouters&listAll=true&page=" + args.page + "&pagesize=" + pageSize + array1.join("") + "&projectid=-1"),
+                                                url: createURL("listRouters&page=" + args.page + "&pagesize=" + pageSize + array1.join("") + "&projectid=-1"),
```

Comment:
```
listall | If  set to false, list only resources belonging to the command's caller; if  set to true - list resources that the caller is authorized to see.  Default value is false
```
so if the admin calls this and a router would not belong to them... I'll test some scenarios